src/scss/1_base/html5.scss

```diff
@@ -16,7 +18,6 @@ time, mark, audio, video {
   padding: 0;
   border: 0;
   font-size: 100%;
-  font: inherit;
```

Comment:
have a look at this, think you should be removing the rule above as would override the font-size property